### PR TITLE
Remove TwoPaneView from tab navigation

### LINKED ISSUE #2150: TwoPaneView should not be included in tab navigation
**Describe the bug**
The TwoPaneView control is currently added automatically to tab-navigation although it shouldn't be for many reasons:
- No user interaction possible
- Its behavior should be similar to all other layout controls (Grid, StackPanel, RelativePanel)

**Steps to reproduce the bug**
- Create a new app
- Add a TwoPaneView control
- Add a Button
- Press many times <kbd>tab</kbd>
--> The focus will switch between the button and the TwoPaneView control

**Expected behavior**
- The focus should stay on the Button

**Version Info**

NuGet package version: 
2.3.200213001

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              | Yes   |
| November 2019 Update (18363)  | Yes   |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 |  Yes  |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |
| 10X                         | Yes |

**Additional context**
I would like to fix it.

### PR DESCRIPTION
**Pull request by rudyhuyn**
## Description
Added `IsTabStop` to the 2 templates of `TwoPaneView` to prevent the control to capture the focus when users use tab navigation.

## Motivation and Context
Fixes #2150 

## How Has This Been Tested?
Tested on Win10 and 10X

### CODE REVIEW

**Comment by chingucoding** [comment]
I think it might be good to add an interaction test for the new tab behavior. A good example is this test:
https://github.com/microsoft/microsoft-ui-xaml/blob/d0f5ad7347598910fb712d441600813237f1c2b3/dev/RatingControl/InteractionTests/RatingControlTests.cs#L463

**Comment by StephenLPeters** [comment]
I've opened #2160 to make the radio buttons pass more reliable so we stop seeing these gates failures.

**Comment by StephenLPeters** [comment]
@rudyhuyn will you be able to add an interaction test for this change?

**Comment by rudyhuyn** [comment]
> @rudyhuyn will you be able to add an interaction test for this change?

Will do it today!

File: dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs [hunk 1]
```diff
@@ -245,27 +246,46 @@ public void PaneLengthTest()
...
+
+        // Verify that tabbing around puts us where we expect
+        [TestMethod]
+        public void VerifyTabKeyWorks()
+        {
+            using (var setup = new TestSetupHelper("TwoPaneView Tests")) // This clicks the button corresponding to the test page.
+            {
+                UIObject twoPaneView = TryFindElement.ByName("TwoPaneView");
+                Verify.IsNotNull(twoPaneView, "TwoPaneView is present");
+                twoPaneView.SetFocus();
```
Comment (chingucoding) [inline dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs]: I get at crash at this line:

Test method Windows.UI.Xaml.Tests.MUXControls.InteractionTests.TwoPaneViewTests.VerifyTabKeyWorks threw exception: 
    System.Runtime.InteropServices.COMException: Error HRESULT E_FAIL has been returned from a call to a COM component.
  Stack Trace: 
    IUIAutomationElement.SetFocus()
    UIObject.SetFocus()
    TwoPaneViewTests.VerifyTabKeyWorks() line 278

File: dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs [hunk 2]
```diff
@@ -146,7 +146,7 @@ public void RegionTest()
             {
                 Log.Comment("Verify horizontal split regions");
                 SetComboBox("SimulateComboBox", "LeftRight");
-
```
Comment (ranjeshj) [inline dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs]: nit: Stray newline

**Comment by chingucoding** [comment]
Looks good to me. 👍

File: dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs (hunk 2 above)
Comment (rudyhuyn) [reply dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs]: Can you give more info?

No new lines have been added/deleted. All of these lines had many space characters and were not consistent with the rest of the code and I suppose we don't want to keep them this way.

**Pull request merged**